src/legacy/core_plugins/kibana/public/visualize/saved_visualizations/saved_visualizations.js

```diff
@@ -36,7 +36,7 @@ savedObjectManagementRegistry.register({
 });
 
 app.service('savedVisualizations', function (SavedVis, Private, kbnUrl, chrome) {
-  const visTypes = Private(VisTypesRegistryProvider);
+  const visTypes = VisTypesRegistryProvider;
```

Comment:
use `visualizations.types.get` (from the new visualizations plugin) ... the get is not there yet i think but it should be added so we can get rid of vistypes registry as soon as possible